.circleci/config.yml

```diff
@@ -0,0 +1,541 @@
+  test_node10_py2_pip9:
+    <<: *defaults
+    docker:
+      - image: node:10
+    steps:
+      - test:
+          pip_ver: '9.0.3'
+          python_ver: '2.7'
+  test_node10_py3_pip10:
+    <<: *defaults
+    docker:
+      - image: node:10
+    steps:
+      - test:
+          pip_ver: '10.0.0'
+          python_ver: '3.6'
+  test_node10_py2_pip10:
+    <<: *defaults
+    docker:
+      - image: node:10
+    steps:
+      - test:
+          pip_ver: '10.0.0'
+          python_ver: '2.7'
+  test_node10_py3_pip18:
+    <<: *defaults
+    docker:
+      - image: node:10
+    steps:
+      - test:
+          pip_ver: '18.1.0'
+          python_ver: '3.6'
+  test_node10_py2_pip18:
+    <<: *defaults
+    docker:
+      - image: node:10
+    steps:
+      - test:
+          pip_ver: '18.1.0'
+          python_ver: '2.7'
+  test_node8_py3_pip9:
+    <<: *defaults
+    docker:
+      - image: node:8
+    steps:
+      - test:
+          pip_ver: '9.0.3'
+          python_ver: '3.6'
+  test_node8_py2_pip9:
+    <<: *defaults
+    docker:
+      - image: node:8
+    steps:
+      - test:
+          pip_ver: '9.0.3'
+          python_ver: '2.7'
+  test_node8_py3_pip10:
+    <<: *defaults
+    docker:
+      - image: node:8
+    steps:
+      - test:
+          pip_ver: '10.0.0'
+          python_ver: '3.6'
+  test_node8_py2_pip10:
+    <<: *defaults
+    docker:
+      - image: node:8
+    steps:
+      - test:
+          pip_ver: '10.0.0'
+          python_ver: '2.7'
+  test_node8_py3_pip18:
+    <<: *defaults
+    docker:
+      - image: node:8
+    steps:
+      - test:
+          pip_ver: '18.1.0'
+          python_ver: '3.6'
+  test_node8_py2_pip18:
+    <<: *defaults
+    docker:
+      - image: node:8
+    steps:
+      - test:
+          pip_ver: '18.1.0'
+          python_ver: '2.7'
+  test_node6_py3_pip9:
+    <<: *defaults
+    steps:
+      - test:
+          pip_ver: '9.0.3'
+          python_ver: '3.6'
+  test_node6_py2_pip9:
+    <<: *defaults
+    steps:
+      - test:
+          pip_ver: '9.0.3'
+          python_ver: '2.7'
+  test_node6_py3_pip10:
+    <<: *defaults
+    steps:
+      - test:
+          pip_ver: '10.0.0'
+          python_ver: '3.6'
+  test_node6_py2_pip10:
+    <<: *defaults
+    steps:
+      - test:
+          pip_ver: '10.0.0'
+          python_ver: '2.7'
+  test_node6_py3_pip18:
+    <<: *defaults
+    steps:
+      - test:
+          pip_ver: '18.1.0'
+          python_ver: '3.6'
+  test_node6_py2_pip18:
+    <<: *defaults
+    steps:
+      - test:
+          pip_ver: '18.1.0'
+          python_ver: '2.7'
+  build:
+    <<: *defaults
+    steps:
+      - checkout_and_merge
+      - run:
+          name: Use snyk-main npmjs user
+          command: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> .npmrc
+      - run:
+          name: Install dependencies
+          command: npm install
+      - run:
+          name: Run tests
+          command: npm run build
+  release:
+    <<: *defaults
```

Comment:
travis has following note on release stage:

`node_js: "8" # This *has* to be the "build leader"`

we're using node:6 here - is that important?

Reply:
> # This *has* to be the "build leader"

I originally wrote that.. it's fine, you can ignore it (was just a travis + semantic-release quirk)

Reply:
oops sorry see you've already answered similar question from Daniel
yea think until we drop node 6 this make sense.